ee/ucp/admin/install/install-on-azure.md

```diff
@@ -4,25 +4,107 @@ description: Learn how to install Docker Universal Control Plane in a Microsoft
 keywords: Universal Control Plane, UCP, install, Docker EE, Azure, Kubernetes
 ---
 
+Docker UCP closely integrates into Microsoft Azure for its Kubernetes Networking 
+and Persistent Storage feature set. UCP deploys the Calico CNI provider for data path networking and leverages Azure CNI for IP address management. There are infrastructure prerequisites that are required prior to UCP installation for the Azure IPAM functionality.
```

Comment:
please switch `Azure CNI for IP address management` to `Azure IPAM for IP address management`